invenio_app_ils/ui/src/invenio_app_ils/common/components/ResultsTable/ResultsTableBody.js

```diff
@@ -1,50 +1,59 @@
 import React, { Component } from 'react';
 import PropTypes from 'prop-types';
 import { Button, Table } from 'semantic-ui-react';
+import isBoolean from 'lodash/isBoolean';
+import isNumber from 'lodash/isNumber';
 
 export default class ResultsTableBody extends Component {
-  renderCell = (cell, column, id, colIndex) => {
-    return (
-      <Table.Cell key={colIndex + '-' + id} data-test={column + '-' + id}>
-        {cell}
-      </Table.Cell>
-    );
+  renderValue = value => {
```

Comment:
Why not having the formatters knowing how to format the values? Until now, the ResultsTable was not "caring" about how to render a value, but it was the responsability of the formatter.
I think, IMHO, it was more correct before with the formatter setting Yes or No (or maybe in the future we want a green tick or red cross)...

WDYT?